content/episodes/2018-07-16-S03E02-press-play.md

```diff
@@ -0,0 +1,367 @@
+---
+categories: "episodes"
+date: 2018-07-16T09:00:00-07:00
```

Comment:
We should verify the intended go-live time. I originally had 9am on the last episode and apparently that was later than it should have been and we had to hastily update it on the morning of (https://github.com/mozilla/irlpodcast/commit/80768d3f87c841606424a4bb89d22bae34da59ad). I think this should be 7am but we should confirm with Michaela. (I just asked on slack, awaiting a response)